dbt_platform_helper/constants.py

```diff
@@ -3,7 +3,7 @@
 # Todo: Can we get rid of this yet?
 PLATFORM_HELPER_VERSION_FILE = ".platform-helper-version"
 # Todo: Move to ???
-DEFAULT_TERRAFORM_PLATFORM_MODULES_VERSION = "5"
+DEFAULT_TERRAFORM_PLATFORM_MODULES_VERSION = "7"
```

Comment:
Do we have version 7 of TPM?

Reply:
@chiaramapellimt , Version 7 is to be released, that's why this is part of a breaking change.
Version ^ will be the work on the IP filter spoofing that's still waitinmg to be merged.
Then this PR and John's work on the codebase pipelines will be in version 7.

It's so that we don't over burden services with lots of breaking changes at once I believe, this keeps the new logic seperate.

Reply:
Is it a breaking change for `platform-tools` when we update the main default version of `terraform-platform-modules`?

Reply:
Yes, it should be actually. It will force users with pipelines to use a later breaking version of t-p-m

Reply:
@antroy-madetech Might be better placed to answer that